scripts/python-skywater-pdk/skywater_pdk/fet_simulator/generate-docs-plots.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright 2020 SkyWater PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+from pathlib import Path
+import sys
+import contextlib
+import traceback
+import errno
+
+from fet_simulator import generate_fet_plots
+
```

Comment:
This module needs a proper doc string (which you can also use in the help).

Reply:
I added docstrings for:
```
scripts/python-skywater-pdk/skywater_pdk/simulation/analog/fet.py
scripts/python-skywater-pdk/skywater_pdk/simulation/analog/generate-fet-bins-plots.py
```